distribution/docs/src/main/resources/content/_running/starting-intro.adoc

```diff
@@ -226,3 +226,68 @@ If the system fails to start automatically on boot with any messages about the J
 
 Because ${branding} is built on top of Apache Karaf, more information on operating ${branding} can be found in the http://karaf.apache.org/index/documentation.html[Karaf documentation].
 
+
+==== Automatically Starting Solr as a Process
```

Comment:
❓ Why can't the DDF service script also start solr? Can we at least include the necessary files so you don't need internet access to configure the system?

Reply:
❗️ According to @rzwiefel everything necessary is included in the distro, so these instructions need to be corrected to refer to the included files instead.

Reply:
I am with you here. I feel we are moving away from the fact that even when we started DDF as a service, it was all inclusive. Now the customer has to go setup its own service instance for Solr when they might not actually care to do so.

Reply:
Updated to reference the distribution provided solr only.